src/Payum/Stripe/Constants.php

```diff
@@ -9,6 +9,12 @@ class Constants
 
     const STATUS_FAILED = 'failed';
 
+    const PAYUM_STRIPE_APP_NAME = "Payum Stripe";
+
+    const PAYUM_STRIPE_VERSION = "1.4.0";
```

Comment:
I think this rather needs to use [Composer\InstalledVersions::getVersion](
https://getcomposer.org/doc/07-runtime.md#knowing-the-version-of-package-x), so that we don't have to update this constant with each new release.